src/plugin/regions/index.js

```diff
@@ -127,11 +127,15 @@ export default class RegionsPlugin {
         });
         this.wavesurfer.Region = Region;
 
+        const scrollWidthProportion = 0.05;
```

Comment:
ideally, add comment why it's 0.05 or near that value

Reply:
I didn't actually add that 0.05 to the code, just modified a line that contained it already, so I don't 100% know why it was used. But I can take a stab at it.